# Post relative to covisible keyframes
FrameInfo frame
std_msgs/ByteMultiArray pose